scripts/deploy/watchdog-config/user-management-compose.yml

```
@@ -0,0 +1,8 @@
+services:
+
+  prometheus:
+    volumes:
+      - ./user-management/user-management-prometheus.yml:/etc/prometheus/scrape_configs/user-management-prometheus.yml:ro
+  grafana:
+    volumes:
+      - ./user-management/user-management-grafana.json:/etc/grafana/provisioning/dashboards/user-management/user-management-grafana.json:ro
```

This is the path that ended up working for me:

```suggestion
      - ./user-management/user-management-grafana.json:/etc/grafana/provisioning/dashboards/CHT/user-management-grafana.json:ro
```